### DISCUSSION TITLE
Continually Rejected by the pre-check

### DISCUSSION THREAD

**Discussion by DefNotRaph** [community/community#42583] [GitHub Education]
I am a distance learner and I keep getting rejected by the precheck for the craziest reasons. I submitted an enrollment letter and used a valid school email. I have no idea why I cant even get my aplication processed.

Errors:
* Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.,Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?,You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
* Your image should be one of the following:
* Dated school ID

**Comment by jfullstackdev** [comment]
it's the right place you are posting but there is no assurance that there will be a reply from the Staff,

you can try also Support 

https://support.github.com/contact/account

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159
